src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -357,37 +357,35 @@ export const evidenceSummaryView = ({ formData }) => {
   );
 };
 
-const FullNameViewField = ({ formData }) => {
-  const { first, middle, last, suffix } = formData;
-  return <strong>{first} {middle} {last} {suffix}</strong>;
-};
-
-const SsnViewField = ({ formData }) => {
-  const ssn = formData.slice(5);
-  const mask = <span>•••-••-</span>;
-  return <p>Social Security number: {mask}{ssn}</p>;
-};
-
-const VAFileNumberViewField = ({ formData }) => {
-  const vaFileNumber = formData.slice(5);
-  const mask = <span>•••-••-</span>;
-  return <p>VA file number: {mask}{vaFileNumber}</p>;
-};
-
-const DateOfBirthViewField = ({ formData }) => {
-  return <p>Date of birth: <DateWidget value={formData} options={{ monthYear: false }}/></p>;
+/**
+ * @param {ReactElement|ReactComponent|String} srIgnored -- Thing to be displayed visually,
+ *                                                           but ignored by screen readers
+ * @param {String} substitutionText -- Text for screen readers to say instead of srIgnored
+ */
+const srSubstitute = (srIgnored, substitutionText) => {
+  return (
+    <div style={{ display: 'inline' }}>
+      <span aria-hidden>{srIgnored}</span>
+      <span className="sr-only">{substitutionText}</span>
+    </div>
+  );
 };
 
-const GenderViewField = ({ formData }) => <p>Gender: {genderLabels[formData]}</p>;
-
-export const veteranInformationViewField = (formData) => {
+export const veteranInformationViewField = (data) => {
+  const { ssn, vaFileNumber, dateOfBirth, gender } = data;
+  const { first, middle, last, suffix } = data.fullName;
+  const mask = srSubstitute('●●●–●●–', 'ending with');
   return (
     <div>
-      <FullNameViewField formData={formData.fullName}/>
-      <SsnViewField formData={formData.socialSecurityNumber}/>
-      <VAFileNumberViewField formData={formData.vaFileNumber}/>
-      <GenderViewField formData={formData.gender}/>
-      <DateOfBirthViewField formData={formData.dateOfBirth}/>
+      <p>This is the personal information we have on file for you.</p>
+      <div className="blue-bar-block">
```

Comment:
I like how descriptive this is, but wonder whether it is consistent with our class naming conventions, which seem to typically favor describing usage over appearance.

Reply:
Yeah, I'm not fond of this name, but never looped back to change it because I didn't know what else to call it that wasn't similarly generic. If you have suggestions, I'm all ~ears~ eyes. :smiley:

Reply:
something along the lines of `address-block` seems helpful? it could be qualified as `disability-address-block` to make it form specific if that is of interest.